badge/README.md

```diff
@@ -0,0 +1,154 @@
+# Badge components
+
+## Badge design specs
+
+You can find the design specs on [decathlon.design](https://www.decathlon.design/).
+
+## Usage
+
+If you want to use components of this module in your android mobile application, you should
+first add the Gradle dependency in your Gradle file:
+
+```kotlin
+implementation("com.decathlon.vitamin.compose:badge:<versions>")
+```
+
+### VitaminCountBadges
+
+```kotlin
+object VitaminCountBadges {
+    @Composable
+    fun Accent(
+        count: Int,
+        badgeSize: BadgeSize = VitaminBadgeSizes.smallSize(),
+        badgeStyle: BadgeStyle = VitaminBadgeStyle.accentStyle(badgeSize = badgeSize),
```

Comment:
I didn't check the implementation yet but I'm afraid to see that we need to pass the `BadgeSize` object here.

Reply:
I used badgeSize, badgeColor and textStyle (which need badge size to update textSize) instead. But if you think is not a good idea, perhaps we can rework it together